pyomo/contrib/incidence_analysis/interface.py

```diff
@@ -388,22 +522,36 @@ def get_adjacent_to(self, component):
             List of constraint or variable data objects adjacent to the
             provided component
 
+        Example
+        -------
+        >>> import pyomo.environ as pyo 
+        >>> from pyomo.contrib.incidence_analysis import IncidenceGraphInterface
+        >>> m = pyo.ConcreteModel()
+        >>> m.x = pyo.Var([1, 2]) 
+        >>> m.eq1 = pyo.Constraint(expr=m.x[1]**2 == 7)
+        >>> m.eq2 = pyo.Constraint(expr=m.x[1]*m.x[2] == 3)
+        >>> m.eq3 = pyo.Constraint(expr=m.x[1] + 2*m.x[2] == 5)
+        >>> igraph = IncidenceGraphInterface(m)
+        >>> adj_to_x2 = igraph.get_adjacent_to(m.x[2])
+        >>> print([c.name for c in adj_to_x2])
+        ['eq2', 'eq3']
```

Comment:
You should add the `.. doctest::` command in this docstring so that this short example is tested (as long as this docstring is included in API documentation on RTD)